pyproject.toml

```diff
@@ -4,10 +4,9 @@ version = "0.1.0"
 description = ""
 authors = ["Jonathen Sundy <jonathensundy@gmail.com>"]
 readme = "README.md"
-package-mode = false
 
 [tool.poetry.dependencies]
-python = "^3.12"
+python = "^3.10.14"
```

Comment:
Why is python being downgraded?

Reply:
The latest pytorch docker image is on python 3.10.4. If we dont use this docker image then we have to deal with installation of cuda and all that other hectic stuff.